lib/segment/tests/integration/sparse_discover_test.rs

```diff
@@ -138,6 +138,7 @@ fn sparse_index_discover_test() {
                 storage_type: VectorStorageType::Memory,
                 index: Indexes::Plain {},
                 quantization_config: None,
+                multivec: None,
```

Comment:
Because we have a lot of default values here, I propose to use `..Default::default()` everywhere in this PR for all default fields

Reply:
I am not sure about this TBH.
It is annoying but useful to see all the places impacted by a new field.